docs/general/Arhitecture.md

```diff
@@ -4,6 +4,7 @@
 
 ## Components
 
+- **[CLI commands](/docs/general/CLI.md)** - overview of CLI commands
```

Comment:
Does this really go into Architecture.md. 
Would be nice to think about this a bit a link everything from Home.md.